tests/test_association_task.py

```diff
@@ -221,11 +323,10 @@ def _store_dia_objects_and_sources(self):
         # to them. The DIASources are "observed" in g and r.
 
         # Create an empty database
-        assoc_db_config = AssociationDBSqliteConfig()
-        assoc_db_config.db_name = self.db_file
-        assoc_db_config.filter_names = self.filter_names
-        assoc_db = AssociationDBSqliteTask(config=assoc_db_config)
-        assoc_db.create_tables()
+        ppdb = Ppdb(config=self.ppdbConfig,
+                    afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                     DiaSource=make_dia_source_schema()))
+        ppdb._schema.makeSchema()
```

Comment:
Given the discussion on lsst/dax_ppdb#5, this line should be `ppdb.makeSchema()` but otherwise doesn't need to be changed (or it could be merged with the similar code in the test case just above it).

Reply:
Ditto with above.